src/groupByKey.ts

```diff
@@ -2,6 +2,20 @@ type GroupsMap<T> = {
   [key: string]: T[];
 };
 
-export function groupByKey(items, key) {
-  // write code here;
+type T = object;
```

Comment:
you dont need this
check how generics works
https://www.typescriptlang.org/docs/handbook/2/generics.html